features/createroom/api/src/main/kotlin/io/element/android/features/createroom/api/ui/AvatarActionListView.kt

```diff
@@ -16,7 +16,7 @@
 
 @file:OptIn(ExperimentalMaterialApi::class)
 
-package io.element.android.features.createroom.impl.configureroom.avatar
+package io.element.android.features.createroom.api.ui
```

Comment:
It looks like these and the other files that have gone in `createroom.api` are shared between `createroom.impl` and `roomdetails.impl`.
Though I'm not sure whether they belong to `createroom.api` or to a shared UI module (matrix-ui or else?) since they don't carry any presenter or business logic with them. It feels odd that feature X has to import the feature `createroom` just to get access to these shared pieces of UI (but not to the `createroom` feature itself).

Furthermore, should we name this with a `ModalBottomSheet` suffix instead of `ListView`? It is indeed a bottom sheet.

Reply:
Good call. I've moved them to `matrixui` and renamed `ListView` to `BottomSheet`.